### ISSUE TITLE
Icons gone?!

### ISSUE THREAD

**Issue by LasseRafn** [googlearchive/js-marker-clusterer#54]
The default cluster-icon is suddenly gone?

**Comment by ignaciojcano** [comment]
Samething happened to me, the default URL is still poiting to the googlecode svn repository...

Workaround is to serve them from your own site or a cdn and modify the image path in the constructor
`{imagePath:'<path-to-images->'}`

Something like this `{imagePath:'http://example.com/components/js-marker-clusterer/images/m'}`

**Comment by hubson-bropa** [comment]
not cool. need the images? they're in the source, https://github.com/googlemaps/js-marker-clusterer/tree/gh-pages/images

**Comment by powermail** [comment]
also please make sure to make the image path relative and not absolute as image giving error on https

**Comment by naume** [comment]
pls add the images ASAP again 
https://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclusterer/images/m2.png
https://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclusterer/images/m1.png

**Comment by yannbug** [comment]
+1

**Comment by davidpelayo** [comment]
No plans on recovering back resources even though have been migrated to github? It's causing many issues in thousand of apps. I consider they shouldn't have been deleted.

By the way, there are reactions to support comments, instead of just writing useless `+1`. They are cleaner from the conversation development perspective.

**Comment by LasseRafn** [comment]
Thanks! :-)

However as I was running production, I had to find a quick fix. I downloaded the assets (m1-m5) and setup the styles accordingly. If they are 100% correct, I don't know. (please correct me if this is wrong; I'm not sure about the **anchor**)

```
var mc = new MarkerClusterer(map, markers, {
    styles: [{
        url: '/images/maps/m1.png',
        height: 53,
        width: 53,
        anchor: [26, 26],
        textColor: '#000',
        textSize: 11
    }, {
        url: '/images/maps/m2.png',
        height: 56,
        width: 56,
        anchor: [28, 28],
        textColor: '#000',
        textSize: 11
    }, {
        url: '/images/maps/m3.png',
        height: 66,
        width: 66,
        anchor: [33, 33],
        textColor: '#000',
        textSize: 11
    }, {
        url: '/images/maps/m4.png',
        height: 78,
        width: 78,
        anchor: [39, 39],
        textColor: '#000',
        textSize: 11
    }, {
        url: '/images/maps/m5.png',
        height: 90,
        width: 90,
        anchor: [45, 45],
        textColor: '#000',
        textSize: 11
    }]
});
```

**Comment by ghost** [comment]
@LasseRafn This has been working for me:

```
var clustererOptions = {
    styles: [
        {
            textColor: 'white',
            url: 'css/images/markerclusterer/m1.png',
            height: 52,
            width: 53
        },
        {
            textColor: 'white',
            url: 'css/images/markerclusterer/m2.png',
            height: 55,
            width: 56
        },
        {
            textColor: 'white',
            url: 'css/images/markerclusterer/m3.png',
            height: 65,
            width: 66
        },
        {
            textColor: 'white',
            url: 'css/images/markerclusterer/m4.png',
            height: 77,
            width: 78
        },
        {
            textColor: 'white',
            url: 'css/images/markerclusterer/m5.png',
            height: 89,
            width: 90
        }
    ]
};
```

Note the dimensions (width, height). I didn't have to set the anchor.

**Comment by stivaugoin** [comment]
+1 
Thanks @WACMemphis

**Comment by PeterTheOne** [comment]
fixed by 2f659a3

**Comment by eliasZX32** [comment]
show!!! Thank you very much!!!
